test/test.expected

```diff
@@ -1266,34 +1266,35 @@ groupoid-laws.cooltt:128.7-128.11 [Info]:
     assoc A {_x₂ => p _x₂} ⊥ {_x₂ => q _x₂} ⊥ {_x₂ => r _x₂} j _x₁
 
 --------------------[hcom-type.cooltt]--------------------
-hcom-type.cooltt:17.8-17.13 [Info]:
+hcom-type.cooltt:16.3-16.8 [Info]:
   Emitted hole:
     _x : [ ⊤ ]
     i : 𝕀
     _x₁ : [ i = 0 ∨ i = 1 ]
     |- ?asdf : nat
 
 
-hcom-type.cooltt:17.16-17.17 [Info]:
+hcom-type.cooltt:16.11-16.12 [Info]:
   Emitted hole:
     _x : [ ⊤ ]
     i : 𝕀
-    |- ? : (_x₁ : [ i = 0 ]) → nat
+    _x₁ : [ i = 0 ]
+    |- ? : nat
     
     Boundary:
-    i = 0 ∨ i = 1
-    |- _x₁ => [ ⊤ => ?asdf _x 0 * ]
+    ⊤
+    |- ?asdf _x 0 *
 
 
-hcom-type.cooltt:17.18-17.19 [Info]:
+hcom-type.cooltt:16.13-16.14 [Info]:
   Emitted hole:
     _x : [ ⊤ ]
     i : 𝕀
     |- ? : nat
     
     Boundary:
     i = 0 ∨ i = 0 ∨ i = 1
-    |- i = 0 => ?asdf _x 0 *
+    |- i = 0 => #3 _x 0 *
```

Comment:
In fact, I wonder: why do we have `i=0 \/ i=0` in here? @favonia

Reply:
`i=0 \/ i=0` was not simplified because we currently do not have semantic simplification that would "maintain" the shape of the input. This could be another PR for kado. Like `simplify_human`? LOL

Reply:
@jonsterling I have no ideas about `?asdf` and don't have time today to check.

Reply:
@favonia I am not too worried about the `?asdf` because it is at least sound. haha